packages/js-drive/lib/abci/errors/BlockExecutionContextNotFoundError.js

```diff
@@ -0,0 +1,16 @@
+const grpcErrorCodes = require('@dashevo/grpc-common/lib/server/error/GrpcErrorCodes');
+
+const AbstractAbciError = require('./AbstractAbciError');
+
+class BlockExecutionContextNotFoundError extends AbstractAbciError {
```

Comment:
We don't need to make it abci. Just normal DriveError and it will be converted to InternalError. Please check others